server-protocol.md

```diff
@@ -104,6 +114,45 @@ was sent in the wrong order), an `error` response will be sent, This response
 will include the error string in the `error` key, and a full copy of the
 original message dictionary in `orig`.
 
+
+## Permission and Abilities
```

Comment:
I think not mentioning this as "abilities" would be better. We kind of got rid of it in this iteration anyways. Same for the last paragraph.

Reply:
okay, got rid of "abilities" (just "permission" now).